src/set_theory/ordinal_arithmetic.lean

```diff
@@ -2298,6 +2242,7 @@ begin
   { exact (mul_is_normal ho).apply_omega }
 end
 
+local infixr ^ := @pow ordinal ordinal ordinal.has_pow
```

Comment:
Do you really need a local infixr for just one theorem?

Reply:
I tried to remove it, but the proof failed due to rewrite troubles, so I'd say we do need it.